eigrpd/eigrp_hello.c

```diff
@@ -405,8 +405,8 @@ void eigrp_hello_receive(struct eigrp *eigrp, struct ip *iph,
 			   inet_ntoa(nbr->src));
 }
 
-uint32_t FRR_MAJOR;
```

Comment:
This is a uint32_t, not a integer

Reply:
Ok, fixing it. I that change instead of the SCNu32 because it avoided breaking the scanf line :-)